pipeline-runner/tests/testthat/test-qc-7-embed_and_cluster.R

```diff
@@ -571,7 +571,7 @@ test_that("download_cl_metadata_file loads cl_metadata tables correctly", {
 
   local_mock_cl_metadata_table(cl_metadata, "mock_experiment_id")
   res <- stubbed_download_cl_metadata_file(config)
-  withr::defer(unlink(file.path(".", basename(config$metadataS3Path))))
+  withr::defer(unlink(file.path(".", basename(config$metadata_s3_path))))
```

Comment:
add test for not running clustering when the flag is there.

Reply:
Added test for `clustering_should_run = FALSE`